DyCore/locale/de/informatron.cfg

```diff
@@ -238,17 +238,13 @@ menu_bonus=Boni / RPG
 title_bonus=Boni / RPG
 page_bonus=[color=red]IN ARBEIT, wird vor der Veröffentlichung hinzugefügt[/color]
 
-menu_global_stat=Spielstatistiken
-title_global_stat=Spielstatistiken
-page_global_stat=[color=red]IN ARBEIT, wird vor der Veröffentlichung hinzugefügt[/color]
-
-menu_personal_stat=Persönliche Statistiken
-title_personal_stat=Persönliche Statistiken
-page_personal_stat=[color=red]IN ARBEIT, wird vor der Veröffentlichung hinzugefügt[/color]
-
-menu_attribute=Attribute
-title_attribute=Attribute
-page_attribute=[color=red]IN ARBEIT, wird vor der Veröffentlichung hinzugefügt[/color]
+menu_stats=Persönliche Statistiken
+title_stats=Persönliche Statistiken
+page_stats_1=[color=red]WORK IN PROGRESS, will be added before release[/color]
+page_stats_2=[color=red]WORK IN PROGRESS, will be added before release[/color]
+page_stats_3=[color=red]WORK IN PROGRESS, will be added before release[/color]
+page_stats_4=[color=red]WORK IN PROGRESS, will be added before release[/color]
+page_stats_5=[color=red]WORK IN PROGRESS, will be added before release[/color]
```

Comment:
@MaxiderZocker do you want to translate these as well?

Reply:
Oh Shoot. Missed those. Will do them when I have time.

Reply:
should be fixed now